### ISSUE TITLE
Cannot program v3.0.8

### ISSUE THREAD

**Issue by goran-mahovlic** [f32c/tools#22]
We need to add new board to supported boards

**Comment by mjosaarinen** [comment]
Yup ran into this with some new boards and was puzzled why a "cable" is found only if I downgrade the "eprom version" from 3.0.8 to 3.0.3. The error messages could be better.

**Comment by goran-mahovlic** [comment]
We have fork of this repository that is continuing development, and 3.0.8. is added there along with other fixes ...

https://github.com/kost/fujprog

**Comment by tcal-x** [comment]
Yes, I hit this using LiteX for ULX3S.   

I was able to get it to work by adding the 12K v3.0.8 entry.

And yes, `fujprog` worked with no modification.

dmesg for my board shows "Product: ULX3S FPGA 12K v3.0.8".    

`ujprog` is the programmer used by LiteX for this board.
Should I request that Litex start using `fujprog` instead?
